lua/mini/extra.lua

```diff
@@ -384,6 +384,41 @@ MiniExtra.pickers.buf_lines = function(local_opts, opts)
   return H.pick_start(items, { source = default_source }, opts)
 end
 
+--- Colorscheme picker
+---
+--- Pick and load from colorschemes.
+--- Notes:
+--- - Preview temporarily loads a colorscheme
+--- - Canceling reverts to original colorscheme
+---
+---@param local_opts __extra_pickers_local_opts
+---   Not used at the moment.
+---@param opts __extra_pickers_opts
+---
+---@return __extra_pickers_return
+MiniExtra.pickers.colorschemes = function(local_opts, opts)
+  local pick = H.validate_pick('colorschemes')
+
+  local original_bg = vim.o.background
+  local has_colors, colors = pcall(require, 'mini.colors')
+  local selected_cs = has_colors and colors.get_colorscheme() or vim.g.colors_name
```

Comment:
When using the `mini.nvim` repository, `mini.colors` is always present. Perhaps it is better to only use `mini.colors` if it has been setup. Otherwise, `mini.colors` always parses the current `colorscheme`, which seems unnecessary.

```suggestion
  local selected_cs = MiniColors ~= nil and MiniColors.get_colorscheme() or vim.g.colors_name
```